subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -890,6 +974,7 @@ static void cis_disabled_cb(void *param)
 			ll_rx_put(node_terminate->hdr.link, node_terminate);
 			ll_rx_sched();
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
```

Comment:
CIS pause/resume is not only a peripheral feature. Central uses it too.

Reply:
reverted.